package exception;

public class demoexception1 {

	public static void main(String[] args) {
		System.out.println("hello from main method");
		System.out.println("chalega");
		int div=10/0;
		System.out.println(div);
		System.out.println("main() ends here becouse next is closiing curly bracket of main metohd ");

	}

}
